classes/ValidateConstraintTranslator.php

```diff
@@ -49,14 +49,32 @@ public function __construct(TranslatorInterface $translator)
      */
     public function translate($validator)
     {
-        if ($validator === 'isName' || $validator === 'isCustomerName') {
+        if ($validator === 'isName') {
             return $this->translator->trans(
                 'Invalid name',
                 [],
                 'Shop.Forms.Errors'
             );
         }
 
+        if ($validator === 'isCustomerName') {
+            return $this->translator->trans(
+                'Invalid name',
+                [],
+                'Shop.Forms.Errors'
+            ) . PHP_EOL .
+            $this->translator->trans(
+                'The following characters are not accepted: / \ ^ * `',
+                [],
+                'Shop.Forms.Errors'
+            ) . PHP_EOL .
+            $this->translator->trans(
+                'The characters . and 。 are allowed only if they are directly followed by a space or the end of the string.',
```

Comment:
I wonder about the "。" character... it seems pretty specific, is it an absolute necessity to mention it?

Reply:
Well this character is indeed forbidden, but maybe you're right If someone tries to use this character it is probably in order to use the breach so maybe it's not useful to indicate it for usual users